sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3180,6 +3180,20 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
     }
 
   }
+
+  test("SPARK-28670 " +
+    "create function should throw AnalysisException if the UDF class is not found") {
```

Comment:
nit: `  test("SPARK-28670 create function should throw AnalysisException if the UDF class is not found") {`